src/fparser/readfortran.py

```diff
@@ -692,8 +691,8 @@ def next(self, ignore_comments = False):
             # self.show_message(message, sys.stderr)
             # traceback.print_exc(file=sys.stderr)
             logger.debug(''.join(('Traceback\n',''.join( traceback.format_stack()))))
-            logger.critical(red_text('STOPPED READING'))
-            # self.show_message(red_text('STOPPED READING'), sys.stderr)
+            logger.critical('STOPPED READING')
```

Comment:
This line is not covered.

Reply:
Not sure how I'd test this since it is a catch all for unexpected errors. If I worked out something which would cause an unexpected error I should fix it, not use it to test this.

Reply:
Agreed but this is where `monkeypatch` is your friend...